internal/eval/store_eval.go

```diff
@@ -323,6 +323,97 @@ func evalSETEX(args []string, store *dstore.Store) *EvalResponse {
 	return evalSET(newArgs, store)
 }
 
+// Key, start and end are mandatory args.
+// Returns a substring from the key(if it's a string) from start -> end.
+// Returns ""(empty string) if key is not present and if start > end.
+func evalGETRANGE(args []string, store *dstore.Store) *EvalResponse {
+	if len(args) != 3 {
+		return &EvalResponse{
+			Result: nil,
+			Error:  diceerrors.ErrWrongArgumentCount("GETRANGE"),
+		}
+	}
+
+	key := args[0]
+	obj := store.Get(key)
+	if obj == nil {
```

Comment:
it looks like we will return "" even if the start and end values are invalid. This is a deviation from the expected behavior for `GETRANGE`. Could you please check on this and add this test case in integration and unit tests. Thanks

Reply:
If the start and end  are invalid then we throw the following error "ERR value is not an integer or out of range". The above mention case is when an object with the given key is not present in the store

Reply:
Yes, but even in that case, we must not return a result but an error